sql/src/main/java/io/crate/operation/udf/UserDefinedFunctionMetaData.java

```diff
@@ -42,11 +42,13 @@
     private String name;
     private List<FunctionArgumentDefinition> arguments;
     DataType returnType;
+    private List<DataType> argumentDataTypes;
     String language;
     String definition;
 
     public UserDefinedFunctionMetaData(String name,
                                        List<FunctionArgumentDefinition> arguments,
+                                       List<DataType> argumentDataTypes,
```

Comment:
Seems like the only usage of this constructor is doing the same thing as the other constructor overload is doing - so this could probably be removed?